### DISCUSSION TITLE
Error related to Mail app -

### DISCUSSION THREAD

**Discussion by elephantastyczny** [nextcloud/all-in-one#3998] [Questions] [answered]
By looking at Nextcloud-AIO-nextcloud container logs I've noticed a plenty of:

sh: /var/www/html/custom_apps/mail/vendor/christophwurst/kitinerary-bin/src/../bin/kitinerary-extractor: not found
sh: flatpak: not found
sh: kitinerary-extractor: not found

As it seems a compilation issue, I'm wondering if this can be fixed in AIO?
https://github.com/nextcloud/mail/issues/7899

**Comment by szaimen** [comment, marked as answer]
Hi, I fear this actually needs to be fixed in the mail app. So https://github.com/nextcloud/mail/issues/7899 is the correct place.
